README.md

```diff
@@ -6,30 +6,16 @@
   <img src="https://github.com/ForestAdmin/lumber/blob/master/assets/lumber-logo.png?raw=true" alt="Lumber logo">
 </p>
 
-Lumber is a CLI tool that helps you create your backend application in minutes.
-At its core, Lumber has been designed with a modular architecture that leverages a set of plugins.
+Lumber is the CLI tool used to install [Forest Admin](https://www.forestadmin.com), the Admin Panel framework.
+Forest Admin has been designed with scalability in mind to fit requirements from small projects to mature companies.
 
-For example, it takes one command to get a REST API, a GraphQL API or a fully operational Admin Panel.
+## Getting started
 
```

Comment:
Maybe a bit too "austère".. How about: "Get your admin panel running in minutes by [installing Forest Admin](https://app.forestadmin.com/signup)."  ?